docs/app/pages/components/components-sections/notifications/notifications/notifications.component.html

```diff
@@ -0,0 +1,142 @@
+<ux-notification-list direction="above" position="top-right"></ux-notification-list>
+
+<div class="btn-container">
+    <button class="btn button-primary m-r-sm" (click)="notificationService.show(notification,
+        { duration: duration, backgroundColor: backgroundColor })">Show Notification</button>
+    <button class="btn button-secondary m-r-sm" (click)="notificationService.dismissAll()">Dismiss Notifications</button>
+</div>
+
+<ng-template #notification let-notificationRef>
+
+    <div class="dismiss">
+        <div class="hpe-icon hpe-close" (click)="notificationService.dismiss(notificationRef)"></div>
+    </div>
+    
+    <div class="notification-wrapper">
+
+        <div class="notification-icon">
+            <span class="hpe-icon hpe-chat"></span>
+        </div>
+    
+        <div class="notification-content">
+            <h4 class="notification-title">Messages Received</h4>
+            <p class="notification-description">You have 16 messages</p>
+            <p class="notification-timestamp">{{ notificationRef.date | date:'EEEE, MMMM d, y, h:mm:ss a' }}</p>
+        </div>
+
+    </div>
+    
+</ng-template>
+
+<hr>
+
+<div class="row uxd-customize-example">
+    <div class="col-md-12">
+        <accordion>
+            <accordion-group heading="Customize Example...">
+                <div class="row uxd-customize-row">
+                    <div class="col-md-4 col-sm-12">
+                        <label for="duration">Duration</label>
+                        <ux-number-picker [(value)]="duration" min="0" max="3600000"></ux-number-picker>
+                    </div>
+                </div>
+                <div class="form-group">
+                    <label class="form-label">Background Color</label>
+                    <div class="input-group m-b-sm">
+                        <input type="text" placeholder="Enter color" class="form-control" [(value)]="backgroundColor" text-string-color>

```

Comment:
Remove `text-string-color` - this is an AngularJS directive with no effect here.

Also binding to `value` does not work here, you need to use `ngModel` instead.